docs/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "formidable-oss-landers-docs",
+  "version": "0.1.0",
+  "main": "static.config.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "react-static start",
+    "clean": "rimraf dist",
+    "build": "react-static build",
+    "prettier-all": "npx prettier --write '{src,static.config,scripts}/**/*.{js,jsx}'",
```

Comment:
GLOBAL: I don't think we need to use `npx` for any actual dependencies we include.